app/models/backend_api.rb

```diff
@@ -34,6 +34,16 @@ class BackendApi < ApplicationRecord
 
   scope :orphans, -> { where.has { id.not_in(BackendApiConfig.selecting { :backend_api_id }) } }
 
+  scope :without_service, (lambda do |param_service_id|
```

Comment:
```suggestion
  scope :without_service, ->(service_id) do
```

Reply:
Then RuboCop would complain

Reply:
with the `->` i mean, because it is multiline

Reply:
Then I disagree with Rubocop. The way I suggested is more readable IMO.

Reply:
It complains about multiline when we use `{..}` instead of `begin..end` I think.

Reply:
It still complains :disappointed: